### PR TITLE
Merge Network release branch into master

### PR DESCRIPTION
**Pull request by anton-evseev**
This is `network-may-release` branch plus additional commits:
- [Resolve merge conflict in cSpell.json](https://github.com/Azure/azure-rest-api-specs/pull/6632/commits/7c89fec7247a2f82d656c863fb37c1d76a08a797)
- [jsonfmt](https://github.com/Azure/azure-rest-api-specs/pull/6632/commits/9f64bebf0d5294da30d10adf51ac5048972fa769)
- [Fix examples to pass ModelValidation](https://github.com/Azure/azure-rest-api-specs/pull/6632/commits/1eb692082bc8ab8b984b8a66952ed57846b0d51a)

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/examples/ApplicationGatewayAvailableSslOptionsGet.json [hunk 1]
```diff
@@ -4,6 +4,142 @@
     "subscriptionId": "subid"
   },
   "responses": {
-    "200": {}
+    "200": {
+      "body": {
+        "name": "default",
+        "id": "/subscriptions/subid/resourceGroups//providers/Microsoft.Network/ApplicationGatewayAvailableSslOptions/default",
```
Comment (nschonni) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/examples/ApplicationGatewayAvailableSslOptionsGet.json]: Think the `resourceGroup//` needs to be removed from these ids. Another 16 hits below
```suggestion
        "id": "/subscriptions/subid/providers/Microsoft.Network/ApplicationGatewayAvailableSslOptions/default",
```

Comment (number213) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/examples/ApplicationGatewayAvailableSslOptionsGet.json]: That's what I got from the server
image

Comment (nschonni) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/examples/ApplicationGatewayAvailableSslOptionsGet.json]: Odd, the url in the spec is `/subscriptions/{subscriptionId}/providers/Microsoft.Network/applicationGatewayAvailableSslOptions/default"`

Comment (number213) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/examples/ApplicationGatewayAvailableSslOptionsGet.json]: Possible error on server's side. I'll try to follow up with AppGw team

**Review by MikhailTryakhov — APPROVED** [review]

**Comment by MyronFanQiu** [comment]
@lmazuel  @zikalino Hi~ Do you know how to restart the build process for the Python SDK?

**Comment by zikalino** [comment]
@MyronFanQiu no, I don't know, and I am worried it's going to fail again :-(

**Comment by MyronFanQiu** [comment]
@AutorestCI rebuild python

**Comment by MyronFanQiu** [comment]
Seems not working. @lmazuel Hope to hear from you.

**Pull request merged**